etw/com.py

```diff
@@ -1,140 +0,0 @@
-########################################################################
```

Comment:
Where did this move to?

Reply:
This did not need to be in the project after refactoring. It was removed.